src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    public String getSalaryInfo(String[] names, String[] data,
+                                String dateFrom, String dateTo) {
+        return getDrawResultSalaries(names, data, dateFrom, dateTo);
+    }
+
+    private String getDrawResultSalaries(String[] names, String[] data,
+                                         String dateFrom, String dateTo) {
+        StringBuilder result = new StringBuilder();
+
+        result.append("Report for period ").append(dateFrom).append(" - ")
+                .append(dateTo).append(System.lineSeparator());
+        for (int i = 0; i < names.length; i++) {
+            result.append(names[i]).append(" - ")
+                    .append(getSalarySumEmployee(names,data,dateFrom,dateTo)[i]);
+            if (i != names.length - 1) {
+                result.append(System.lineSeparator());
+            }
+        }
+        return result.toString();
```

Comment:
`if` block is not required if we move separator to beginning of the line
```suggestion
        result.append("Report for period ").append(dateFrom).append(" - ").append(dateTo);
        for (int i = 0; i < names.length; i++) {
            result.append(System.lineSeparator())
                    .append(names[i])
                    .append(" - ")
                    .append(getSalarySumEmployee(names, data, dateFrom, dateTo)[i]);
        }
        return result.toString();
```